jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -421,8 +440,50 @@ public void onClick( ClickEvent event ) {
 
         controlBulkOperations();
     }
+    
+    private Column<ProcessInstanceSummary, ProcessInstanceSummary> initErrorCountColumn() {
+        
+        Column<ProcessInstanceSummary, ProcessInstanceSummary> column = new Column<ProcessInstanceSummary, ProcessInstanceSummary>(
+                new AbstractCell<ProcessInstanceSummary>(){
+
+            @Override
+            public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilder sb) {
+                int errCount =  value != null ? value.getErrorCount() : 0;
+                ProcessRuntimeCSS css = ProcessRuntimeResources.INSTANCE.css();
+                //TODO: set link URL to correct error view perspective path
+                String linkToErrorView = "#Error View?processInstanceId=" + value.getProcessInstanceId();
+                String cssClass = "label-" + (errCount > 0 ? "danger" : "default");
+                String elementId = DOM.createUniqueId();
+                sb.appendHtmlConstant("<a id=\"" + elementId + "\" href=\"#\" class=\"label " + css.processInstanceErrorCount() +
+                        " " + cssClass + "\" data-toggle=\"popover\" data-html=\"true\" data-placement=\"right\"" +
+                        " data-content=\"" + constants.ErrorCountNumber(errCount) + " | <a href='" + linkToErrorView +
+                        "' title='" + constants.ErrorCountViewLink() + "'>" + constants.ErrorCountViewLink() + "</a>\"" +
+                        " data-container=\"body\" data-trigger=\"click\">" + errCount + "</a>");
```

Comment:
The problem I found when using `hover` is that the popover is hidden when the cursor leaves the popover area. Hence it's not possible to click on the "View All" link which is in the popover panel, but is located outside of the popover area.
I'll try with `focus`